libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -114,6 +114,16 @@ public static void AddBotRuntime(this IServiceCollection services, IConfiguratio
 
         internal static void AddBotRuntimeSkills(this IServiceCollection services, IConfiguration configuration)
         {
+            // If TenantId is specified in config, add the tenant as a valid JWT token issuer for Bot to Skill conversation.
+            // The token issuer for MSI and single tenant scenarios will be the tenant where the bot is registered.
+            var validTokenIssuers = new List<string>();
+            var tenantId = configuration.GetSection(MicrosoftAppCredentials.MicrosoftAppTenantIdKey)?.Value;
+            if (!string.IsNullOrWhiteSpace(tenantId))
+            {
+                validTokenIssuers.Add($"https://sts.windows.net/{tenantId}/"); // For MSI auth, 1.0 token
```

Comment:
If we have these strings in AuthenticationConstants, let's use those. If not, maybe add these to MSIAuthenticator or authentication constants? Also, if there are docs for these issuers, would be great to link in the comments so that future devs know where this information came from.

Reply:
Do these constants need to be configurable, with a fallback to the constant (as was done in the CloudAdapter work)? Or, will these urls work for isolated clouds, or does that matter?

Reply:
Added the constants to `AuthenticationConstants`. @EricDahlvang - these constants don't need to be configurable. They are the list of issuers we validate against today, but for botframework.com tenant. This change just configures tenant id to be the bot's home tenant.